src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -282,6 +282,60 @@ p, li { white-space: pre-wrap; }
             </layout>
            </widget>
           </item>
+          <item>
+           <widget class="QGroupBox" name="wineGroupBox">
+            <property name="title">
+             <string>Wine/Proton system calls (requires DWARF unwinding)</string>
+            </property>
+            <layout class="QVBoxLayout" name="verticalLayout">
+             <property name="spacing">
+              <number>0</number>
+             </property>
+             <property name="leftMargin">
+              <number>4</number>
+             </property>
+             <property name="topMargin">
+              <number>4</number>
+             </property>
+             <property name="rightMargin">
+              <number>4</number>
+             </property>
+             <property name="bottomMargin">
+              <number>4</number>
+             </property>
+             <item>
+              <widget class="QRadioButton" name="wineNoneRadioButton">
+               <property name="toolTip">
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;With Wine 7 (and Proton 7), the implementation of Windows system calls operate on a different stack. Orbit's DWARF stack unwinding requires a copy of the stack to successfully unwind the callstack offline. This copy will not contain the &amp;quot;Windows user-space&amp;quot; stack, and thus unwinding will fail on the bridge method (__wine_syscall_dispatcher).&lt;/p&gt;&lt;p&gt;With this option, Orbit will not perform any special handling for Wine system calls.&lt;/p&gt;&lt;p&gt;On Wine 7, Orbit will likely fail unwinding after the &amp;quot;__wine_syscall_dispatcher&amp;quot; function and report an unwinding error.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Can we make __wine_syscall_dispatcher monospace or bold in the two places where it appears (and in any case, consistent)?

Reply:
Also in the other tooltips.

Reply:
I can make it bold.